src/components/auth-components/signin.js

```diff
@@ -127,21 +167,31 @@ export default function CenteredGrid() {
                         <img src={logo} className={classes.logo} alt="logo"></img>  
                         <Typography className={classes.heading2} variant="h4">New Here!?</Typography>
                         <Typography className={classes.subheading2} variant="h5">Start your journey with us! <br></br>Sign up now</Typography>
-                        <a href="#" className={classes.signInLink}>
+                        <a href="/signup" className={classes.signInLink}>
                             Sign Up
                         </a> 
                     </Grid>
                     <Grid item className={classes.gridLeft} md={8} xs={12}>
-                        <Typography className={classes.heading} variant="h3">Sign In <br></br>to Seretonin Inn</Typography>
-                        <a href="#">
+                        <Typography className={classes.heading} variant="h3">Sign In <br></br>to Seroton-Inn</Typography>
+                        <a href="/auth/google">
                             <img src={gButton} className={classes.logo} alt="google button"></img>
                         </a> 
-                        <Typography className={classes.subheading}>or use your email account</Typography>
+                        <Typography className={classes.subheading}>or Sign in the old-fashioned way</Typography>
                         <Container maxWidth="sm">
                             <form noValidate autoComplete="off">
-                                <TextField id="outlined-basic" className={classes.inputText} label="Email" variant="outlined" />
-                                <TextField id="outlined-basic" className={classes.inputText} label="Password" variant="outlined" />
-                                <Button className={classes.formButton} variant="contained">Sign In</Button>
+                                <TextField id="outlined-basic" 
+                                className={classes.inputText} 
+                                label="Username" 
+                                value={info.username}
+                                onChange={handleChange('username')}
+                                variant="outlined" />
+                                <TextField id="outlined-basic" 
+                                className={classes.inputText} 
+                                label="Password" 
+                                value={info.password}
+                                onChange={handleChange('password')}
```

Comment:
Are validations added for password and cpassword

Reply:
Validations for password will be added in the next commit.